code/modules/projectiles/guns/ballistic/automatic.dm

```diff
@@ -321,7 +321,7 @@
 
 /obj/item/gun/ballistic/automatic/sniper_rifle
 	name = "sniper rifle"
-	desc = "A long ranged weapon that does significant damage. No, you can't quickscope."
+	desc = "A long ranged anti-materiel rifle based on the legendary M107. No, you can't quickscope."
```

Comment:
I feel like we generally try to shy away from namedropping real guns, takes away from the immersion a little bit. Also to be nitpicky, I feel like it's odd to use the US army's designation for the rifle rather than the actual name.

Reply:
M107 or M82, it's the same rifle. Not to mention the sprite is quite literally identical to one.

Reply:
Please read my comment again?